cmd/codegen/generator/go/generator.go

```diff
@@ -158,7 +158,8 @@ func (g *GoGenerator) GenerateClient(ctx context.Context, schema *introspection.
 		)
 
 		// Get the go package from the module
-		pkg, _, err := loadPackage(ctx, "/module")
+		// We assume that we'll be located at the root source directory
+		pkg, _, err := loadPackage(ctx, ".")
```

Comment:
:eyes: do we not need to set a `WithWorkdir` correspondingly in `GenerateClient` as well then?

Reply:
We do :)

```
		dagql.Selector{
			Field: "withWorkdir",
			Args: []dagql.NamedInput{
				{
					Name:  "path",
					Value: dagql.NewString(filepath.Join(goSDKUserModContextDirPath, rootSourcePath)),
				},
			},
		},
```